src/physics/cam/clubb_intr.F90

```diff
@@ -4586,17 +5934,25 @@ subroutine clubb_tend_cam( state,   ptend_all,   pbuf,     hdtime, &
         !  called, the shallow convective mass flux will ALWAYS be zero, ensuring that this cloud
         !  fraction is purely from deep convection scheme.
```

Comment:
Seems like this comment and the one below should be updated to reflect shalcu is included when do_clubb_mf_rad is true.

Reply:
The MF part of CLUBB really should be using the deep infrastructure, however implementing that should be a separate PR as it requires a lot of conversions to pressure based mass fluxes used by ZM, among other things. For now I'll just add comments that CLUBB+MF is using parts of the shallow infrastructure, but not cmfmc_sh.